src/cursor_manager.ts

```diff
@@ -365,8 +390,9 @@ export class CursorManager implements Disposable, NeovimRedrawProcessable, Neovi
             `${LOG_PREFIX}: Updating cursor pos in neovim, winId: ${winId}, pos: [${pos.line}, ${pos.character}]`,
         );
         const vimPos = [pos.line + 1, pos.character]; // nvim_win_set_cursor is [1, 0] based
-        const request: [string, unknown[]][] = [["nvim_win_set_cursor", [winId, vimPos]]];
-        await callAtomic(this.client, request, this.logger, LOG_PREFIX);
+        await this.client.request("nvim_win_set_cursor", [winId, vimPos]); // a little faster
```

Comment:
Is this measurably faster, or you just think it will be because there are a little fewer stack calls?

Reply:
I did a simple test and it took 2ms less on average using request.

Reply:
Cool, delete commented lines